packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -283,10 +283,25 @@ const DropDownEditor = TextBox.inherit({
             .eq(0);
     },
 
+    _getHaspopupAria() {
+        return 'true';
+    },
+
+    _getAutocompleteAria() {
+        const { searchEnabled, acceptCustomValue, readOnly, disabled } = this.option();
+
+        const isEditable = acceptCustomValue || searchEnabled;
+        const isDisabled = readOnly || disabled;
+
+        const autocompleteAria = isEditable && !isDisabled ? 'list' : 'none';
+
+        return autocompleteAria;
+    },
```

Comment:
We also need to consider a scenario in SelectBox when input substitution is rendered on typing. I believe this attr should be different in this case. 
You can look for `_shouldSubstitutionBeRendered` method in SelectBox.